ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -493,6 +493,72 @@ - (BOOL)isAccessibilityElement {
 @end
 
 
+#pragma mark - ORKFormItemConfirmTextCell
+
+@implementation ORKFormItemConfirmTextCell
+
+- (void)setSavedAnswers:(NSDictionary *)savedAnswers {
+    [super setSavedAnswers:savedAnswers];
+    
```

Comment:
To be cautious, throw exception on  `savedAnswers` is nil, at here or in the super class?